packages/plugin-commands-installation/src/yarnUtil.ts

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
maybe let's make a subdirectory for the import command and put this file there.

and add yourself as codeowner of it to `.github/CODEOWNERS`

```
/packages/plugin-commands-installation/src/import/ @fireairforce 
```

Reply:
ok, thx, i will do this